packages/core/src/blocks/core/group/index.js

```diff
@@ -0,0 +1,106 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+
+
+/**
+ * WordPress Dependencies
+ */
+const { addFilter } = wp.hooks;
+const { Fragment } = wp.element;
+const { createHigherOrderComponent } = wp.compose;
+
+const allowedBlocks = [ 'core/group' ];
+
+import Inspector from './controls';
+
+export const addGroupFilters = ( settings ) => {
+
+  function addAttributes(settings) {
+    if ( allowedBlocks.includes( settings.name ) ) {
+      settings.attributes = Object.assign( settings.attributes, {
+        contentAlignment: {
+          type: 'string',
+          default: 'full'
+        },
+      } );
+    }
+
+    return settings;
+  }
+
+  const withControls = createHigherOrderComponent((BlockEdit) => {
+    return (props ) => {
+      return(
+        <Fragment>
+          <BlockEdit {...props} />
+          { props.isSelected && allowedBlocks.includes( props.name ) && <Inspector { ...{ ...props } } /> }
```

Comment:
If feels more legible to me if you would first check `allowedBlocks.includes( props.name )` and if not, simply return the BlockEdit component. Also my feeling is `props.isSelected` is not needed in this case since it should be the default behaviour for elements wrapped inside `InspectorControls`